pkgs/build-support/agda/default.nix

```diff
@@ -70,8 +72,8 @@ let
         installPhase = if installPhase != null then installPhase else ''
           runHook preInstall
           mkdir -p $out
-          find \( ${concatMapStringsSep " -or " (p: "-name '*.${p}'") (extensions ++ extraExtensions)} \) -exec cp -p --parents -t "$out" {} +
-          runHook postInstall
+          find -not \( -path ${everythingFile} -or -path ${interfaceFile everythingFile} \) -and \( ${concatMapStringsSep " -or " (p: "-name '*.${p}'") (extensions ++ extraExtensions)} \) -exec cp -p --parents -t "$out" {} +
+  runHook postInstall
```

Comment:
This also looks indented badly but it might just be github

Reply:
Use Refined GitHub. It marks you spaces with dots and tabs with arrows.

Reply:
I fixed the indentation.